client/app/assets/styles/media-queries.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
   // so we must add those manually to the output.
   '--medium-viewport': `(min-width: ${medium})`,
   '--large-viewport': `(min-width: ${large})`,
+  '--small-and-medium-viewport': `(max-width: ${large})`,
```

Comment:
I think both the `--large-viewport` and `--small-and-medium-viewport` queries apply when the viewport is exactly `large`, which is probably not what we want?